### DISCUSSION TITLE
Cannot update app

### DISCUSSION THREAD

**Discussion by MaxMatti** [immich-app/immich#10429] [General]
### The bug

I had to update the server because the app update to version 106 was force pushed to my wifes phone but the Google Play store does not offer me the update so now I can't login because my app is still on version 105.

### The OS that Immich Server is running on

docker-compose

### Version of Immich Server

106.1

### Version of Immich Mobile App

105

### Platform with the issue

- [X] Server
- [ ] Web
- [X] Mobile

### Your docker-compose.yml content

```YAML
Not applicable
```

### Your .env content

```Shell
Not applicable
```

### Reproduction steps

```bash
1.
2.
3.
...
```

**Comment by mmomjian** [comment]
If the play store is doing a staggered rollout, we don’t have any control over that. You can install the APK from the GitHub releases.
